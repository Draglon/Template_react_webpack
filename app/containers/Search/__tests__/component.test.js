import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SearchComponent from '../component';

describe('<SearchComponent /> container', () => {
  const props = {};
  let component;

  it('render property - with search query', () => {
    component = shallow(<SearchComponent {...props} />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
