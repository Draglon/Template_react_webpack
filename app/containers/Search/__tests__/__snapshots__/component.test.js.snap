// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchComponent /> container render property - with search query 1`] = `
<Row
  className="top-padding"
  gutter={0}
  type="flex"
>
  <Col
    lg={
      Object {
        "offset": 4,
        "span": 16,
      }
    }
    md={
      Object {
        "offset": 3,
        "span": 18,
      }
    }
    sm={
      Object {
        "offset": 2,
        "span": 20,
      }
    }
    xl={
      Object {
        "offset": 5,
        "span": 14,
      }
    }
    xs={
      Object {
        "offset": 1,
        "span": 22,
      }
    }
  >
    <Search
      enterButton="Search"
      onSearch={[Function]}
      placeholder="Enter movie name"
      size="large"
    />
  </Col>
</Row>
`;
